### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by jdbs9514**
- Setup your project:
  - Create a new directory and initialize it with git.
  - Use `npm init -y` command to create package.json file.
  - Create the entry point for your JavaScript code called `index.js`
  - Create `index.css` file for your styles.
  - Create the main `index.html` file. Link your `index.js` and `index.css` files there.
  - Create a directory called `modules`.
- Re-create your _Awesome books_ app following the rules:
  - Use ES6 syntax.
  - Divide your code into modules. Save each module in a separate file in the `modules` dir. Import modules in the `index.js` file. For this exercise - the more modules the better!
  - Keep all funcionalities of the app without errors.
  - Refactor your methods and functions to arrow functions.
  - Use let and const in a correct way.

 
- Display the current date below the navigation bar. 
  - For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
  - Download the script and save it to your project. 
  - Import it as a module.
  - See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+## Author
+
+👤 **Josué Benavides**
+- twitter:@JODA9514
+- Gmail:jdbs9514@gmail.com
+- GitHub: @jdbs9514
+
+## 📝 License
+
+This project is MIT.md licensed.
```
Comment (Bondok6) [inline README.md:31]: - [OPTIONAL] I suggest adding `MIT.md` file to your project 👌 [Microverse Template](https://github.com/microverseinc/readme-template)

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    </header>
+    <main>
+        <section class="section-table active">
+            <h1 class="title">Awesome Books</h1>
+            <table id="table" class="table">
+                <thead>
+                    <tr>
+                        <td></td>
+                        <td></td>
+                        <!--<td><button type="button" class="remove">Remove</button></td>-->
```
Comment (Bondok6) [inline index.html:35]: [OPTIONAL] Kindly, remove unused code to follow the [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and to make your project clean 👌💯

File: index.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  addLink, addNew, contact, contactLink, list, listLink,
+} from './modules/action.js';
+
+const dateTime = document.getElementById('date');
+let btnRemove;
+let newBook;
+let dataBase = [];
+
+// create the date //
+dateTime.innerHTML = DateTime.now();
```
Comment (Bondok6) [inline index.js]: - [OPTIONAL] Great job using `Luxon` library 💯👌 I would like to give you a small tip to display the date in a nicer way I suggest using `toLocaleString(DateTime.DATETIME_MED)` method 😄
> For example 
```
const dateT = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
dateTime.innerHTML = dateT;
```

image

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @jdbs9514  🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 3 above)
Comment (jdbs9514) [reply index.js]: thanks, i really use.

**Pull request merged**